extensions/cdc-debezium/src/main/java/com/hazelcast/jet/cdc/impl/ChangeRecordImpl.java

```diff
@@ -102,26 +98,15 @@ public String toString() {
         return toJson();
     }
 
-    @Override
-    public int getFactoryId() {
-        return CdcJsonDataSerializerHook.FACTORY_ID;
-    }
-
-    @Override
-    public int getClassId() {
-        return CdcJsonDataSerializerHook.CHANGE_RECORD;
-    }
-
-    @Override
-    public void writeData(ObjectDataOutput out) throws IOException {
+    void writeData(ObjectDataOutput out) throws IOException {
         out.writeUTF(keyJson);
         out.writeUTF(valueJson);
     }
 
-    @Override
-    public void readData(ObjectDataInput in) throws IOException {
-        keyJson = in.readUTF();
-        valueJson = in.readUTF();
+    static ChangeRecordImpl readData(ObjectDataInput in) throws IOException {
+        String keyJson = in.readUTF();
```

Comment:
this method can be inlined into the serializer

Reply:
The fields used are implementation details (to some extent) and I didn't want to break the private access, but I guess package access might work. This is why I came up with these methods and I used them everywhere for consistency. Will inline them.